experiments/data/prepare_mixed_data.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import json
+
+import datasets
+
+from chemnlp.data_val.config import DataMixingConfig
+from chemnlp.utils import load_config
+
+RANDOM_SEED = 1234
+
+
+def run(config_path: str) -> None:
+    """Create a mixed dataset for a given YAML defined configuration"""
+    raw_config = load_config(config_path)
+    config = DataMixingConfig(**raw_config)
+
+    mixed_data = datasets.interleave_datasets(
+        datasets=[
+            datasets.load_from_disk(data_path) for data_path in config.data_paths
+        ],
+        probabilities=config.data_proportions,
+        seed=RANDOM_SEED,
+        stopping_strategy="first_exhausted",
```

Comment:
Maybe we expose this `stopping_strategy` option as a configuration variable. I think we might have use cases where we want to oversample a smaller dataset because it's more important to us or something, what do you think?

Reply:
Thanks, thats a good idea :)